<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-0I845IR8" failures="1" tests="1" name="Assesertion_Demo.Soft_Assertion_Class" time="0.008" errors="0" timestamp="2024-08-16T14:44:18 IST" skipped="0">
  <testcase classname="Assesertion_Demo.Soft_Assertion_Class" name="softAssertionTestMethod" time="0.008">
    <failure message="The following asserts failed:
	This method will pass only when the condition become true expected [true] but found [false],
	This method will return the pass when when expected reult matched with actual result expected [9] but found [8],
	This method will pass only when both actual and expected are not match did not expect [50] but found [50],
	This method will compare the addresses of mentioned 2 objects and pass only when 2 addresses are macthing expected [1] but found [java.lang.Object@7bd7d6d6],
	This method will retunr pass only when specified object has null expected [null] but found [str]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
This method will pass only when the condition become true expected [true] but found [false],
This method will return the pass when when expected reult matched with actual result expected [9] but found [8],
This method will pass only when both actual and expected are not match did not expect [50] but found [50],
This method will compare the addresses of mentioned 2 objects and pass only when 2 addresses are macthing expected [1] but found [java.lang.Object@7bd7d6d6],
This method will retunr pass only when specified object has null expected [null] but found [str]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at Assesertion_Demo.Soft_Assertion_Class.softAssertionTestMethod(Soft_Assertion_Class.java:61)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- softAssertionTestMethod -->
  <system-out/>
</testsuite> <!-- Assesertion_Demo.Soft_Assertion_Class -->
